theme/voidy-bootstrap/templates/includes/custom/open_in_colab_footer.html

```diff
@@ -4,6 +4,10 @@
     var doc0 = document.getElementById("colablink");
     var url = location.href ;
     var article_year = url.slice(33, 37);
+    var article_month = url.slice(38, 40);
```

Comment:
In this moment it's ok. It can make some bugs sometime however. It would be better to find a better way.
(It is ok to do it later. Not necessarily to fix in this pull request.)